README.md

```diff
@@ -161,6 +162,20 @@ Additionally, checkout our [video from JuliaCon 2023](https://www.youtube.com/wa
     <a href="https://www.youtube.com/watch?v=qXrvDVm_fnE"><img width="460" src="https://img.youtube.com/vi/qXrvDVm_fnE/0.jpg"></a>
 </p>
 
+Certainly! Here's an updated version of the roadmap section with added emojis:
+
+# Roadmap
+
+Our high-level project roadmap outlines the key milestones and focus areas over the next few quarters:
+
+| Q1/Q2 2024          | Q3/Q4 2024                | 2025                | 2026                                      |
+|---------------------|---------------------------|--------------------|-------------------------------------------|
+| 🧩 **Nested models with [GraphPPL.jl](https://github.com/biaslab/GraphPPL.jl)**     | 🛡️ **Robustness (NaN, Inf free)** | 🌐 **Stochastic Processes** | 🔄 **Automated inference with non-exponential family** 
+| 🔄 **Automated inference with [ExponentialFamily.jl](https://github.com/biaslab/ExponentialFamily.jl)**                 | 🧠 **Memory-efficiency** | 🚀 **Efficient message passing on a huge graph** | 📊 **Inference over graph structure**
```

Comment:
Regarding _Efficient message passing on a huge graph_, isn't this already one of the key selling points of RxInfer? Do you rather mean something like "Resource-adaptive inference"?

Reply:
Big graphs, not huge ones :) 
Anyway, it was supposed to be a triggering mechanism, but it's pretty slang. Resource-adaptive inference may sound better.